python/tank/authentication/shotgun_shared/__init__.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2015 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+"""
```

Comment:
note: having talked to @patrick-hubert-adsk yesterday sounds like the goal was to keep things separate and shared across RV and toolkit. I think it would be great to discuss the pros/cons with separate/shared and the right structure in each case!